assets/src/dashboard/app/views/shared/telemetryBanner.js

```diff
@@ -0,0 +1,175 @@
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+/**
+ * Internal dependencies
+ */
+import { TypographyPresets } from '../../../components';
+import { Close as CloseSVG } from '../../../icons';
+import { ICON_METRICS } from '../../../constants';
+import useTelemetryOptIn from './useTelemetryOptIn';
+
+const Banner = styled.div.attrs()`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 161px;
+  margin: 0 20px;
+  background-image: url('/wp-content/plugins/web-stories/assets/images/analytics-banner-bg.png');
+  border-radius: 8px;
+`;
+
+const Header = styled.div`
+  width: 100%;
+  text-align: center;
+`;
+
+const Title = styled.h1`
+  display: inline-block;
+  font-size: 18px;
+  font-weight: ${({ theme }) => theme.typography.weight.normal};
+  line-height: 24px;
+  margin-bottom: 13px;
+`;
+
+const Label = styled.label.attrs({ htmlFor: 'telemetry-banner-opt-in' })`
+  display: flex;
+`;
+
+export const LabelText = styled.span`
+  ${TypographyPresets.Small};
+  color: ${({ theme }) => theme.colors.gray400};
+  margin-bottom: 16px;
+  max-width: 530px;
+`;
+
+const VisitSettingsText = styled(LabelText)``;
+
+const CheckBox = styled.input.attrs({
+  type: 'checkbox',
+  id: 'telemetry-banner-opt-in',
+})`
+  height: 18px;
+  width: 18px;
+  margin: 5px 12px 0 0;
+`;
+
+const CloseIcon = styled(CloseSVG).attrs(ICON_METRICS.TELEMETRY_BANNER_EXIT)`
+  color: ${({ theme }) => theme.colors.white};
+  display: flex;
+  justify-content: flex-start;
+  align-items: center;
+`;
+
+const ToggleButton = styled.button.attrs({
+  ['aria-label']: __('Dismiss Notice', 'web-stories'),
+})`
+  border: none;
+  padding: 4px;
+  border-radius: 50%;
+  background: ${({ theme }) => theme.colors.gray200};
+  cursor: pointer;
+  float: right;
+  margin-right: 14px;
+
+  &:hover svg {
+    color: ${({ theme }) => theme.colors.gray50};
+  }
+  &:active svg {
+    color: ${({ theme }) => theme.colors.gray50};
+  }
+`;
+
+export function TelemetryOptInBanner({
+  visible = true,
+  disabled = false,
+  onChange = () => {},
+  onClose = () => {},
+  checked = false,
+}) {
+  return visible ? (
+    <Banner>
+      <Header>
+        <Title>{__('Help improve the editor!', 'web-stories')}</Title>
```

Comment:
According to Figma, this title should change to "Your selection has been updated. Thank you for helping to improve the editor." when the checkbox is toggled. 

Quote: "Please update the message in the banner to provide feedback that selection was reflected. Close the banner after some delay. If the user decides to undo their selection, revert title message back to “Help improve the editor"

Reply:
Woops, totally missed that - on it now!